src/main/java/com/github/manolo8/darkbot/core/entities/Box.java

```diff
@@ -135,4 +141,61 @@ public Beacon(int id, long address) {
             super(id, address);
         }
     }
+
+    public static class FakeBox extends Box implements eu.darkbot.api.game.entities.FakeEntity.FakeBox {
+        private Timer timeout;
+        private long removeDistance;
+        private boolean isRemoveWhenAttemptSelect;
+
+        public FakeBox(String boxName, Location loc, long removeDistance, long keepAlive, boolean isRemoveWhenAttemptSelect) {
+            super(Integer.MIN_VALUE);
```

Comment:
should probably use different IDs for these, could probably just hold a global fake entity id counter and keep on using extremely low numbers (counting up from min value)

Reply:
what about random id? `super(ThreadLocalRandom.current().nextInt(Integer.MIN_VALUE, -1000));`
but I found where it can be used and not sure that it's matter

Reply:
lets not use randomly generated value here, a defined value or add boolean check to tag them as a "fake" to show the entity is as fake

Reply:
> lets not use randomly generated value here, a defined value or add boolean check to tag them as a "fake" to show the entity is as fake

it's random negative values. For detecting fake can use https://github.com/darkbot-reloaded/DarkBotAPI/pull/91/files#diff-21bb577e67916994b49cca980705fdb72d7f47329f09bac1596e907da599c906R12

Reply:
So there's two issues here: the game itself already uses negative ids for some stuff, starts with -1, then -2, etc, and it likely will reach -1000 if you run the bot for long enough.

Secondly, with random values it's possible to end with a collision, and i'd want to avoid that.

I don't see the issue in simply having `private static int CURR_ID = Integer.MIN_VALUE;` and assigning id as `super(CURR_ID++)`, which should generate unique, sequantial, negative ids for everything.

And yeah, as to what anjesh mentioned, the ID is not a way to determine if they're fake entities, check via instanceOf

Reply:
if use `private static int CURR_ID = Integer.MIN_VALUE;` and `super(CURR_ID++)` than we have `2 147 483 648` negative values so when we create more entities then IDs will be positive. I think it not good thing. 

P.S. I made it like it wrote in FakeNPC class
![image](https://github.com/darkbot-reloaded/DarkBot/assets/52008918/2d16a666-13b0-4e8a-a17d-7dad0fc5f12c)

Reply:
creating over 2 bilions of entities seems unreal for me, even with runtime over a year
year have over 30 milions of seconds, so you have to create ~70 entities every second to fill it up in a year

Reply:
Sorry i actually meant CURR_ID-- not ++

Reply:
implemented